src/core/workload.py

```diff
@@ -99,6 +99,11 @@ def capacity_jbod_json(self):
         """The JBOD capacity JSON."""
         return f"{self.conf_path}/capacityJBOD.json"
 
+    @property
+    def cruise_control_auth(self):
+        """The credentials file."""
+        return f"{self.conf_path}/cruisecontrol.credentials"
```

Comment:
**question:** Why a separate file? No issue with it, just curious. Does the config assume a path?